app/components/standup_meeting/meeting_update_component.rb

```diff
@@ -0,0 +1,15 @@
+# frozen_string_literal: true
+
+class StandupMeeting::MeetingUpdateComponent < ViewComponent::Base
+  with_collection_parameter :standup_meeting
+
+  def initialize(standup_meeting:, content_type:)
+    @content_type = content_type
+    @standup_meeting = standup_meeting
+    @user = standup_meeting.user
```

Comment:
I think because I included the user, n+1 queries should be avoided here? Is this unsafe?

Reply:
if user was included when making the initial call it should be fine. You can always view the server logs and it'll show you the sql calls you are making (and if you are making multiple of the same)

Reply:
cool, figured as much.